tasks/affective_task/utils/timer.py

```diff
@@ -23,6 +26,11 @@ def timer(seconds: int, callbacks: List[Callable], pre_text: str, screen):
             break
         else:
             seconds_left_to_count = 0 if seconds_left_to_count < 0.0 else int(seconds_left_to_count)
+            TIM = seconds_left_to_count
             render_text_center(pre_text + str(seconds_left_to_count + 1), (300, 50), screen, y_offset=-420)
+        
+        if pre_text == "Team: " and Rating == False:
+            if TIM <= DISCUSSION_TIMER:
```

Comment:
You should not modify the `timer` method. Instead, in the `client_affective_task.py`, render the image using `render_image_center` method, then render the text `Discuss` under the image, then call the `timer` method.